uiautomationutilities/perf_tool/Gen3MeasurementsData.cs

```diff
@@ -0,0 +1,33 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace PerfDiffResultMailer
+{
+    public class Gen3MeasurementsData : IEquatable<Gen3MeasurementsData>
```

Comment:
consider using better name. Gen3MeasurementsData -- doesn't provide clear info.

Reply:
Agreed, 
Renamed to MeasurementsData.cs